api/api.go

```diff
@@ -50,7 +50,7 @@ func RunApi() {
 		interfaceName := context.GetHeader("interface")
 
 		var response = "["
-		devices := wsdiscovery.SendProbe(interfaceName, nil, []string{"dn:NetworkVideoTransmitter"}, map[string]string{"dn": "http://www.onvif.org/ver10/network/wsdl"})
+		devices, _ := wsdiscovery.SendProbe(interfaceName, nil, []string{"dn:NetworkVideoTransmitter"}, map[string]string{"dn": "http://www.onvif.org/ver10/network/wsdl"})
```

Comment:
add a //TODO comment for handling the error in the future

Reply:
Comment added.